### DISCUSSION TITLE
How can I use multi-windows search for all windows but the current one ?

### DISCUSSION THREAD

**Discussion by woosaaahh** [folke/flash.nvim#146] [Q&A] [answered]
Hi !

As said in the title, is there a way to do multi-windows search for all windows except the current one ?

Imho, it would have the benefit to allow more labels available for other windows. (less skips)

**Comment by folke** [comment, marked as answer]
Check the `exclude` option in the config

**Reply by woosaaahh** [reply]
I do it right now.

Thanks a lot that was fast !
